src/CommandingOfficers/Strong.java

```diff
@@ -30,29 +30,27 @@ public instantiator()
     {
       super("Strong");
       infoPages.add(new InfoPage(
-          "Commander Strong knows that an army's strength lies in its people.\n" +
-          "Well-positioned soldiers with proper equipment and training are guaranteed to bring results.\n" +
-          "His focus is therefore on achieving this in his own forces, and denying it of his opponent.\n" +
-          "As a result, his transport units and boots on the ground are among the best, and he brings some extra anti-personnel firepower to keep his opponents in check."));
+          "Commander Strong knows that an army's strength lies in its people; soldiers with proper training and equipment are guaranteed to bring results.\n" +
+          "His transport units and boots on the ground are among the best, and he brings some extra anti-personnel firepower to keep his opponents in check."));
       infoPages.add(new InfoPage(
           "Passive:\n" + 
-          "- +"+PASSIVE_INF_BUFF+"% attack for all footsoldiers\n" +
-          "- +"+PASSIVE_ANTI_INF_BUFF+"% attack vs enemy footsoldiers, when attacking\n" +
-          "- +1 movement and +1 capacity for all transports\n" +
-          "- Strong can build Mechs from air/sea ports\n"));
+          "+"+PASSIVE_INF_BUFF+"% attack for all footsoldiers\n" +
+          "+"+PASSIVE_ANTI_INF_BUFF+"% attack vs enemy footsoldiers, when attacking\n" +
+          "+1 movement and +1 capacity for all transports\n" +
+          "Can build Mechs from air/sea ports\n"));
       infoPages.add(new InfoPage(
           "Strongarm ("+StrongArmAbility.STRONGARM_COST+"):\n" +
           "+"+StrongArmAbility.STRONGARM_BUFF+"% attack and defense for all units\n" +
           "+"+StrongArmAbility.STRONGARM_FOOT_BUFF+"% attack for footsoldiers\n" +
           "+2 movement for footsoldiers and APCs\n" +
-          "Strong can build infantry on air/sea ports\n"));
+          "Can build infantry on air/sea ports\n"));
       infoPages.add(new InfoPage(
           "Mobilize ("+MobilizeAbility.MOBILIZE_COST+"):\n" + 
           "+"+MobilizeAbility.MOBILIZE_BUFF+"% attack for all units\n" +
           "+"+MobilizeAbility.MOBILIZE_DEFENSE_BUFF+"% defense for all units\n" +
           "+2 movement for all units\n" +
-          "Refreshes footsoldiers\n" +
-          "Strong can build footsoldiers on cities, industries, and the HQ\n"));
+          "All footsoldiers can move again\n" +
```

Comment:
I feel like "refresh" is maybe a better term:
"Move again" doesn't necessarily imply *act* again (and vice versa)
"Refresh" sounds like a thing that happens instantly on activation, while "can move again" could easily mean "get refreshed upon acting the first time"

Reply:
"Move again" is the verbiage used on the carts, so I'm not worried about it being understood. Plus moving without being able to act just... isn't a thing.
"Refresh" can work too, but it could also just mean to pass around the lemonade.
Potato potato, really.

Reply:
> "Move again" is the verbiage used on the carts, so I'm not worried about it being understood. Plus moving without being able to act just... isn't a thing.

It's not a thing as the game is now, but it's a reasonably common proposed custom unit/CO mechanic... though in general, I think finding other ways to implement it than "you can move again, but not do anything other than wait" would be preferable for clarity/clean design purposes.

> "Refresh" can work too, but it could also just mean to pass around the lemonade.

Hah, but "resupply" (or even rearm/refuel) and "heal" are their own separate and unambiguous terms. If it wasn't meaningful re: game mechanics, it wouldn't be in the description.

Reply:
I think if we ever add the ability to move-shoot-move, we would need to clarify that in the ability description anyway. Anyone who's played AW will expect "unit can move again" to entail also acting, unless you specify "Can move once more after acting"
So I'm super not worried about this being misinterpreted; this seems like another instance of you trying to impose narrow distinctions on similar terms that aren't defined clearly anywhere (and in this case, your new meaning contradicts the commonly-understood one). That said, if you prefer to use "Refresh", I'm not gonna get too bothered over the precise verbiage.

> Hah, but "resupply" (or even rearm/refuel) and "heal" are their own separate and unambiguous terms
"Resupply" and "heal" can already be done independently on the carts; being able to move and not act cannot.